src/Riok.Mapperly/Descriptors/Mappings/MethodMapping.cs

```diff
@@ -60,8 +61,20 @@ ITypeSymbol targetType
 
     protected MethodParameter? ReferenceHandlerParameter { get; private set; }
 
-    public override ExpressionSyntax Build(TypeMappingBuildContext ctx) =>
-        Invocation(MethodName, SourceParameter.WithArgument(ctx.Source), ReferenceHandlerParameter?.WithArgument(ctx.ReferenceHandler));
+    public override ExpressionSyntax Build(TypeMappingBuildContext ctx)
+    {
+        var parameters = Parameters
+            .Zip(ctx.Parameters, (a, b) => (a, b))
+            .Select(x => x.a.WithArgument(x.b))
+            .Cast<MethodArgument?>()
+            .ToArray();
+        return Invocation(
+            MethodName,
+            SourceParameter.WithArgument(ctx.Source),
+            ReferenceHandlerParameter?.WithArgument(ctx.ReferenceHandler),
+            parameters
+        );
```

Comment:
This is probably not needed anymore (also the Wrap10) if we never automatically call methods with additional parameters (`CallableByOtherMappings = false`).